x-pack/plugins/security_solution/public/resolver/store/data/selectors.ts

```diff
@@ -42,38 +44,33 @@ import { formatDate } from '../../view/panels/panel_content_utilities';
 /**
  * If there is currently a request.
  */
-export function isLoading(state: DataState): boolean {
-  return state.pendingRequestDatabaseDocumentID !== undefined;
+export function isTreeLoading(state: DataState): boolean {
+  return state.tree.pendingRequestParameters !== undefined;
 }
 
 /**
- * A string for uniquely identifying the instance of resolver within the app.
+ * If a request was made and it threw an error or returned a failure response code.
  */
-export function resolverComponentInstanceID(state: DataState): string {
-  return state.resolverComponentInstanceID ? state.resolverComponentInstanceID : '';
+export function hadErrorLoadingTree(state: DataState): boolean {
```

Comment:
:grey_question: Suggest `hadErrorLoadingLastTree` to be more precise?

Reply:
I'm not opposed to it. That being said, this code will only try once. If you get an error, you'll have it for good.